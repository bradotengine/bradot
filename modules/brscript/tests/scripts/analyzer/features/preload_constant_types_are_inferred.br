const Constants = preload("brscript_to_preload.notest.br")

func test():
	var a := Constants.A
	print(a)
