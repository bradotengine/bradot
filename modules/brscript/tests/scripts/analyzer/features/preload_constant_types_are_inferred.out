BRTEST_OK
42
